src/Engine/RadiumEngine.hpp

```diff
@@ -165,6 +166,12 @@ class RA_ENGINE_API RadiumEngine
      */
     const std::vector<std::shared_ptr<Core::Asset::FileLoaderInterface>>& getFileLoaders() const;
 
+    /**
+     * Return the aabb of the scene by aggregating aabb from entity.
+     * @return the AABB
```

Comment:
no need for the second line right?

Reply:
This allows doxygen to generate a well formated documentation. The second line is mandatory. The first could be rewritten to be less similar ...

Reply:
Then just use one line with `@returns the aabb of ...`